app/views/devise/registrations/_informable_fields.html.slim

```diff
@@ -32,4 +32,11 @@
         |&nbsp; Male
         br
         = p.radio_button :gender, 'female', class:"radio2"
-        |&nbsp; Female &nbsp;
\ No newline at end of file
+        |&nbsp; Female &nbsp;
+
+javascript:
+  $(document).ready(function(){
+    $('#dob').datetimepicker({
```

Comment:
@GauravMakwana-BTC : Use object literal pattern , let me know if you need any help in it.